pkg/rest/coverage/swagger_analysis.go

```diff
@@ -0,0 +1,119 @@
+package coverage
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/go-openapi/loads"
+	"github.com/go-openapi/spec"
```

Comment:
Similar to above.

Reply:
I prefer to keep as it is, from the pkg context it's easy to predict what it does.